tests/e2e/tests/multicluster/split_horizon_test.go

```diff
@@ -225,10 +227,50 @@ type deployableConfig struct {
 	// (e.g the default global authentication policy) and need to be modified for tests.
 	Removes    []resource
 	applied    []string
+	appliedOut []string
 	removed    []string
 	kubeconfig string
 }
 
+func (c *deployableConfig) Wait(istioctls []*framework.Istioctl) error {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
we should make this a generic functionality somewhere. Doesn't make much sense for each test to have 40 lines of code to wait for config

Reply:
preferable as part of pkg/test/framework given this area is more or less deprecated

Reply:
I agree.  I put it here for the moment, because split_horizon_test seems to duplicate all the Config logic from pilot_test, and the pilot_test logic is used all over the place.  By putting it in Split Horizon first, I can try it out while only impacting the one test...